variables.tf

```diff
@@ -39,10 +39,20 @@ variable "secrets" {
   description = "An object representing the secret to expose to the docker container"
 }
 
-variable "health_check_path" {
-  type        = string
-  default     = "/"
-  description = "Path used to check the health of the container"
+variable "health_check" {
+  type = object({
+    healthy_threshold   = number,
+    interval            = number,
+    path                = string,
+    unhealthy_threshold = number
+  })
+  default     = {
```

Comment:
Did you use `terraform fmt`?

Reply:
This was still the indent when it had a single value. Will change it.